### PR TITLE
Sc 25 constants testcases

### PR DESCRIPTION
**Pull request by akp111**

### CODE REVIEW

File: test/unit/constants.test.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      expect(network).toBe('localflare');
+    });
+
+    test('Should throw error as file does not exist', () => {
+      expect(() => networkFromContextFile('')).toThrowError();
+    });
+  });
+
+  describe('getConfig Testcases', () => {
+    it('should return flare config if network is flare or undefined', () => {
```
Comment (avisingh2) [inline test/unit/constants.test.ts:130]: can you make the `it` statement consistent either start with uppercase or lower and follow the same across the test cases

File: test/unit/constants.test.ts [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      expect(context).not.toBeNull;
+    });
+
+    test('Should throw error as file does not exist', () => {
+      expect(() => contextFile('')).toThrowError();
+    });
+  });
+
+  describe('networkFromContextFile Testcases', () => {
+    test('Should return network for valid file', () => {
```
Comment (avisingh2) [inline test/unit/constants.test.ts:119]: here, the test statement start with uppercase and some places it is lowercase.

File: test/unit/cli.test.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  logAddressInfo,
+  logBalanceInfo,
+  logNetworkInfo,
+  logValidatorInfo
+} from '../../src/cli';
+import { logInfo, log } from '../../src/output';
+import { contextEnv } from '../../src/constants';
+import { Context } from '../../src/interfaces';
+import fixtures from '../fixtures/cli.data';
+jest.mock('../../src/output', () => ({
```
Comment (avisingh2) [inline test/unit/cli.test.ts:14]: add blank line

File: test/unit/addDelegate.test.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import { getAddDelegatorParams, getUnsignedAddDelegator } from '../../src/addDelegator';
+import { contextEnv } from '../../src/constants';
+import { Context } from '../../src/interfaces';
+import fixtures from '../fixtures/addDelegate.data';
+import {tranferFundsFromCtoP} from "../helper/testHelpers"
```
Comment (avisingh2) [inline test/unit/addDelegate.test.ts:5]: add blank line

File: test/helper/testHelpers.ts [hunk 5]
```diff
@@ -33,3 +35,12 @@ export const compareValues = (a: any, b: any): boolean => {
   }
   return a === b;
 };
+
+export const tranferFundsFromCtoP = async () => {
```
Comment (avisingh2) [inline test/helper/testHelpers.ts:39]: please add documentation for all the exported function, interfaces, constants and enums

File: test/helper/testHelpers.ts [hunk 6]
```diff
@@ -1,4 +1,6 @@
 import { BN } from '@flarenetwork/flarejs/dist';
+import { exportTxCP } from '../../src/evmAtomicTx';
+import { contextEnv } from '../../src/constants';
 export const serialize = (bufferData: Buffer[]) => {
```
Comment (avisingh2) [inline test/helper/testHelpers.ts:4]: add blank line above
please add documentation for all the exported function, interfaces, constants and enums

File: src/constants.ts [hunk 7]
```diff
@@ -71,12 +71,11 @@ function getConfig(network: string): NetworkConfig {
   return networkConfig
 }
 
-function context(
+export function context(
```
Comment (avisingh2) [inline src/constants.ts:74]: please add documentation for all the exported function, interfaces, constants and enums

File: src/constants.ts [hunk 8]
```diff
@@ -59,7 +59,7 @@ export function getContext(network: string, publicKey?: string, privateKeyHex?:
   return context(getConfig(network), publicKey, privateKeyHex, privateKeyCB58)
 }
 
-function getConfig(network: string): NetworkConfig {
+export function getConfig(network: string | undefined): NetworkConfig {
```
Comment (avisingh2) [inline src/constants.ts:62]: please add documentation for all the exported function, interfaces, constants and enums

File: src/constants.ts [hunk 9]
```diff
@@ -12,7 +12,7 @@ import {
   decodePublicKey
 } from './utils'
 
-function readContextFile(ctxFile: string): ContextFile {
+export function readContextFile(ctxFile: string): ContextFile {
```
Comment (avisingh2) [inline src/constants.ts:15]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 10]
```diff
@@ -326,7 +326,7 @@ function logNetworkInfo(ctx: Context) {
   log(`assetId: ${ctx.avaxAssetID}`)
 }
 
-async function logValidatorInfo(ctx: Context) {
+export async function logValidatorInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:329]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 11]
```diff
@@ -317,7 +317,7 @@ async function logBalanceInfo(ctx: Context) {
   log(`P-chain ${ctx.pAddressBech32}: ${pbalance} FLR`)
 }
 
-function logNetworkInfo(ctx: Context) {
+export function logNetworkInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:320]: please add documentation for all the exported function, interfaces, constants and enums

File: src/addDelegator.ts [hunk 12]
```diff
@@ -59,7 +59,7 @@ export async function getUnsignedAddDelegator(
   }
 }
 
-async function getAddDelegatorParams(
+export async function getAddDelegatorParams(
```
Comment (avisingh2) [inline src/addDelegator.ts:62]: please add documentation for all the exported function, interfaces, constants and enums

File: src/addValidator.ts [hunk 13]
```diff
@@ -63,7 +63,7 @@ export async function getUnsignedAddValidator(
   }
 }
 
-async function getAddValidatorParams(
+export async function getAddValidatorParams(
```
Comment (avisingh2) [inline src/addValidator.ts:66]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 14]
```diff
@@ -163,7 +163,7 @@ async function contextFromOptions(options: OptionValues): Promise<Context> {
 
 // Network is obtained from context file, if it exists, else from --network flag.
 // This is because ledger does not need a context file
-function networkFromOptions(options: OptionValues): string {
+export function networkFromOptions(options: OptionValues): string {
```
Comment (avisingh2) [inline src/cli.ts:166]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 15]
```diff
@@ -176,7 +176,7 @@ function networkFromOptions(options: OptionValues): string {
   return network
 }
 
-function getOptions(program: Command, options: OptionValues): OptionValues {
+export function getOptions(program: Command, options: OptionValues): OptionValues {
```
Comment (avisingh2) [inline src/cli.ts:179]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 16]
```diff
@@ -189,7 +189,7 @@ function getOptions(program: Command, options: OptionValues): OptionValues {
   return { ...allOptions, network }
 }
 
-function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {
+export function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {
```
Comment (avisingh2) [inline src/cli.ts:192]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 17]
```diff
@@ -298,7 +298,7 @@ export async function initCtxJsonFromOptions(options: OptionValues): Promise<voi
 //////////////////////////////////////////////////////////////////////////////////////////
 // Network info
 
-function logAddressInfo(ctx: Context) {
+export function logAddressInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:301]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 18]
```diff
@@ -307,7 +307,7 @@ function logAddressInfo(ctx: Context) {
   log(`secp256k1 public key: 0x${compressedPubKey}`)
 }
 
-async function logBalanceInfo(ctx: Context) {
+export async function logBalanceInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:310]: please add documentation for all the exported function, interfaces, constants and enums

**Comment by akp111** [comment]
addressed in #17

**Pull request closed without merging**